From fbf0544138af02be4d6dd3085e7ecf98ad060bb4 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Wed, 13 Mar 2024 17:30:02 +0100
Subject: [PATCH] GNUmakefile: use env from PATH

This allows using env from coreutils-native (rather than host env),
which resolves the problem of missing -S option on distributions
with much older coreutils such as Ubuntu 18.04.

Once all autobuilder distros are newer than that, this patch
can be dropped, together with corutils-native dependency.

Upstream-Status: Inappropriate [made for obsolete host distros]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>
---
 GNUmakefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/GNUmakefile b/GNUmakefile
index 1234194..65ba327 100644
--- a/GNUmakefile
+++ b/GNUmakefile
@@ -21,7 +21,7 @@
 ########################################################################
 
 
-SHELL := /usr/bin/env
+SHELL := env
 .SHELLFLAGS := -S bash -Eeuo pipefail -c
 
 
